app/components/channel_list_row/index.tsx

```diff
@@ -68,13 +70,12 @@ export default function ChannelListRow({
             return null;
         }
 
-        const color = selected ? theme.buttonBg : theme.centerChannelColor;
         return (
             <View>
                 <CompassIcon
                     name={selected ? 'check-circle' : 'circle-outline'}
                     size={28}
-                    color={color}
+                    color={selected ? theme.buttonBg : changeOpacity(theme.centerChannelColor, 0.32)}
```

Comment:
what is the impact on other components when applying the color opacity instead of what it was before? (please share screenshots of all affected components)